gulpfile.js

```diff
@@ -335,6 +336,7 @@ function buildExamples(watch) {
   buildExample('pinterest.amp.html');
   buildExample('released.amp.html');
   buildExample('twitter.amp.html');
+  buildExample('soundcloud.amp.html');
```

Comment:
One optional change: Add yourself to `test-example-validation.js` and add `amp-soundcloud` to the error whitelist there. If you don't get to it in this change, please send a follow up.